docs/design/marketing-offer-program.md

```diff
@@ -0,0 +1,250 @@
+# Marketing Offer Program Design Document
+
+## Abstract
+
+Build trust with new users by allowing them to try out the network.
+Increase viral growth of our platform through word-of-mouth marketing. 
+Discover the best client referral offer type during our beta release by testing the performance of different client offering types.
+
+**Free Tier Credit**: A free trial for all users who create an account on the Tardigrade network without using a Referral Offer link.  
+
+**Referral Offer Credit**: An incentive program for motivating existing clients to refer their peers to set up paid accounts on the Tardigrade network. 
+Existing clients give their UUID Referral Link to their friends for their friends to create an account and receive an Invite Offer Credit that is greater than the default Free Tier Credit.
+
+## Background
+
+### REFERRAL LINK
+
+- When a user wants to issue a referral link:
+  An existing client who has an account on one of the Tardigrade Satellites is able to retrieve their unique referral link from their dashboard. They can issue their referral link to their friends in the following ways:
+  - Copy the static link associated to their account with the default invite message from their dashboard 
+  - Twitter Share
+
+- When a referral link is accepted:
+  The invitee can accept their invite to redeem their Referral Offer by going through the Referral Link to create an account on the Satellite they were invited to/through.
+  Upon creating their account they must enter their payment information into the Stripe integration and accept/acknowledge that:
+    If they use up their referral credit before the expiration date they should expect a bill for any amount of activity they acquire beyond what was credited to them
+    When their offer period expires they should expect a bill for any activity on the Tardigrade network going forward billed monthly at unit per hour
+
+### AWARDED CREDIT
+
+- When free credit is awarded:
+It is displayed as the USD value that is equivalent to the storage & bandwidth allotted amount for the credit
```

Comment:
Does this mean that all credits are USD values? Offers are always $x USD?

Reply:
Yes, that's how I understand it too